tests/phpunit/ControllerTests/UserGroupControllerTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\ControllerTests;
+
+use App\Contracts\UserGroupContract;
+use App\Http\Controllers\UserGroupController;
+use Illuminate\Http\Request;
+use Mockery;
+use Tests\TestCase;
+
+class UserGroupControllerTest extends TestCase
+{
+    public $utility;
+    public $UserGroupController;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->utility = Mockery::mock(UserGroupContract::class);
+        $this->UserGroupController = new UserGroupController($this->utility);
+    }
+
+    /**
+     * @test
+     */
+    public function getGroup_returns_group_and_email_from_utility_as_array()
+    {
```

Comment:
you should change the method to `getGroup_returns_group_and_id_from_utility_as_array()`